src/views/AboutPage.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+   <div style="background-color:ghostwhite; overflow-y: auto;">
+      <v-app-bar style="background: linear-gradient(38.92deg, #8198AB 36.61%, #2B435D 102.9%), #4A75AD;">
+         <router-link to="/">
+            <svg class="logo" width="40" height="55" viewBox="0 0 61 70" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Is this the icon? I am thinking if we should put it into a separate component, might be cleaner

Reply:
You are right. Let me organize it.